extensions/emojis-com/eslint.config.js

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "eslint/config"
+import raycastConfig from "@raycast/eslint-config"
+import reactHooks from "eslint-plugin-react-hooks"
+
+export default defineConfig([
+  {
+    ignores: ["**/*.generated.ts"],
+  },
+  ...raycastConfig,
+  reactHooks.configs["recommended-latest"],
+])
```

Comment:
As I mentioned in the last comment. Let's use the Raycast original configuration.